lls/main.c

```diff
@@ -337,6 +394,15 @@ lls_proc(void *arg)
 			 * happen. In fact, we may want to reduce the amount
 			 * of times this is called, since reading the HPET
 			 * timer is inefficient. See the timer application.
+			 *
+			 * Also send gratuitous ARP/ND messages to fulfill
+			 * the LACP requirement.
+			 *
+			 * XXX The LACP requirement could be starved if
+			 * the LLS block receives a lot of requests but
+			 * we are unable to answer them -- i.e. the
+			 * number of requests > 0 for a sustained
+			 * period but we never invoke the TX burst.
```

Comment:
You could modify `process_pkts()` above to return how many packets were transmitted to reset the LACP timer since that it can now skip a cycle. Another opportunity to reset the LACP timer is during a scan of the caches if a packet is transmited.